src/support/str.ts

```diff
@@ -0,0 +1,8 @@
+export const kebab = (str: string): string =>
+    str
+        .replace(/([a-z])([A-Z])/g, "$1-$2")
+        .replace(/[\s_]+/g, "-")
+        .toLowerCase();
```

Comment:
Refer to my comment on both usages of this function. It seems it is not needed and can be safely removed.